chain-impl-mockchain/src/certificate/delegation.rs

```diff
@@ -56,10 +56,16 @@ impl StakeDelegation {
 }
 
 impl Serialize for OwnerStakeDelegation {
+    fn serialized_size(&self) -> usize {
+        let delegation_buf =
+            serialize_delegation_type(&self.delegation, ByteBuilder::new()).finalize_as_vec();
+        delegation_buf.len()
```

Comment:
I think this is basically the same thing the default implementation does, is it not? Not too important though.

Reply:
Generally yes, you agree, but I would prefer to left such implementation only for one purpose - to have a reminder that if we will change the `serialize` implementation, to also update `serialized_size` impl.